politeiad/plugins/usermd/usermd.go

```diff
@@ -10,9 +10,11 @@ const (
 	// PluginID is the unique identifier for this plugin.
 	PluginID = "usermd"
 
-	// Plugin commands
-	CmdAuthor      = "author"      // Get record author
-	CmdUserRecords = "userrecords" // Get user submitted records
+	// CmdAuthor gets record author
```

Comment:
Declaration comments that appear in go docs should be full sentences that use proper punctuation.